dockers/docker-database/Dockerfile.j2

```diff
@@ -32,9 +32,21 @@ RUN sed -ri 's/^(save .*$)/# \1/g;                       \
              s/^logfile .*$/logfile ""/;                 \
              s/^# syslog-enabled no$/syslog-enabled no/; \
              s/^# unixsocket/unixsocket/;                \
+             s%pidfile .*%pidfile /var/run/redis/redis.pid%; \
              s/^client-output-buffer-limit pubsub [0-9]+mb [0-9]+mb [0-9]+/client-output-buffer-limit pubsub 0 0 0/ \
             ' /etc/redis/redis.conf
 
+RUN sed -ri 's%DAEMON_ARGS=.*%DAEMON_ARGS=/etc/`basename ${0}`/`basename ${0}`.conf%;  \
+             s/NAME=.*/NAME=`basename ${0}`/;                                          \
+             s/DESC=.*/DESC=`basename ${0}`/;                                          \
+             s%PIDFILE=.*%PIDFILE=$RUNDIR/`basename ${0}`.pid%;                        \
+             ' /etc/init.d/redis-server
+
+RUN cp /etc/init.d/redis-server /etc/init.d/redis
```

Comment:
>cp [](start = 4, length = 2)

Do you want to `mv` instead? I guess `/etc/init.d/redis-server` will not because of your changes.